dashboard/config/locales/en.yml

```diff
@@ -372,6 +372,7 @@ en:
       course_catalog: "Course Catalog"
       project_gallery: "Projects"
       schools: "Schools"
+      schools: "Districts"
```

Comment:
should the string be 'districts:' instead?

Reply:
Lol whoops! Updating now.